src/api/search/src/search.js

```diff
@@ -103,4 +101,57 @@ const search = async (options) => {
     values: hits.hits.map(({ _id }) => ({ id: _id, url: `${POSTS_URL}/${_id}` })),
   };
 };
-module.exports = { search };
+
+/**
+ * authorAutocomplete allows for querying author autocomplete fields
+ * @param { author } - text to search in author field
+ * @return all the results matching the fields text
+ */
+const authorAutocomplete = async ({ author }) => {
+  const results = {
+    query: {
+      match: {
+        'author.autocomplete': {
+          query: author,
+          operator: 'and',
+        },
+      },
+    },
+    highlight: {
+      fields: {
+        'author.autocomplete': {},
+      },
+    },
+  };
+
+  const {
+    body: { hits },
+  } = await client.search({
+    size: 10000,
```

Comment:
I wonder if there's an upper-limit on what we should return so it's responsive to use in the front-end?  There's probably some guidance on how many results to return for autocomplete queries so that you have useful but small downloads, and it can query and re-render everything fast enough to be useful when typing.

I wonder if we should consider using 10? See https://ux.stackexchange.com/posts/56131/revisions

Reply:
I'd like to get a confirmation on this (please see conversation below)
I don't mind changing it to `10` or something else smaller than `10,000`
I just hope that it is clear that if it is too small we might get fewer author results than what was expected of.

Reply:
Let's begin with this, and reduce it as we learn how it performs.